README.md

```diff
@@ -59,8 +59,8 @@ to `./runner.yml` and overriding relevant properties.
 
 To set up the project run:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
```

Comment:
This step should be removed like in other repositories we are now updating.

Reply:
Fixed.